ts/Core/Renderer/SVG/SVGElement.ts

```diff
@@ -1061,19 +1061,12 @@ class SVGElement implements SVGElementLike {
      * @return {Highcharts.SVGElement}
      *         Return the SVG element for chaining.
      */
-    public css(styles: CSSObject): this {
+    public css(styles?: CSSObject): this {
```

Comment:
Q: I can't find the reason why `styles` can be undefined

Reply:
I changed it because the implementation does `if (styles...` several places, so I assumed the typing was wrong. I'll check tomorrow if we actually have any use case where it is `undefined`. Running the test suite should reveal it if you didn't already try that...

Reply:
It does appear to be used with `undefined`, although this probably should be refactored:

https://github.com/highcharts/highcharts/blob/2105d2b1f28241c03253205ea404ca4852e6bf04/ts/Core/Navigator.ts#L1145

That expression actually passes `false|CSSObject`.

There are also at least two other places (Legend navigation and drilldown module) where non-object arguments are passed, but I think we shouldn't address that in this PR.

Reply:
Thanks for the explanation! I agree, not part of the this PR, however worth to refactor in the future